apps/charterafrica/src/components/ToolCard/ToolCard.js

```diff
@@ -49,6 +61,7 @@ const ToolCard = React.forwardRef(function ToolCard(props, ref) {
                 width: "100%",
                 maxWidth: horizontalDisplay ? 585 : "100%",
                 minWidth: 330,
+                height: horizontalDisplay ? "100%" : 160,
```

Comment:
Is `orientation` like `direction`? I think sticking to `direction` like MUI layout components (`Stack`, `Grid`, etc.) may work best where `direction = [row|column]`